common_features.mk

```diff
@@ -265,7 +265,7 @@ ifneq ($(strip $(BACKLIGHT_ENABLE)), no)
     endif
 endif
 
-VALID_WS2812_DRIVER_TYPES := bitbang pwm spi i2c
+VALID_WS2812_DRIVER_TYPES := bitbang pwm spi i2c bb_twi
```

Comment:
The direction here wont work correctly, as ledbbtwi does not actually implement the ws2812 driver interface.

ledbbtwi.c existing in avr seems fine, though its only being used by one keyboard so maybe a more generic "custom" option is better.

Reply:
Well, this does work.

Specifically, what the bb_twi does is add the ledbbtwi file, and add a define to enable it. 

And the problem with "custom" is that the core ledbbtwi stuff doesn't reside in the ws2812.c, but rgblight.